src/tigerbeetle/main.zig

```diff
@@ -6,7 +6,7 @@ const mem = std.mem;
 const os = std.os;
 const log_main = std.log.scoped(.main);
 
-const build_options = @import("tigerbeetle_build_options");
+const build_options = @import("vsr_options");
```

Comment:
Here, I think ideally we'd use a different options struct -- we only need this for commit hash, and that feels like it doesn't belong to vsr_options. Splitting the options into two feels a bit out of scope.